src/staking-v3/components/data/DataList.vue

```diff
@@ -143,6 +149,44 @@ export default defineComponent({
       () => tiersConfiguration.value.numberOfSlots - dAppTiers.value.dapps.length
     );
 
+    const dappsPerTier = computed<number[]>(() => {
+      const tierCounts: { [key: number]: number } = {};
+
+      // Loop through each dApp in the list and count occurrences per tierId
+      dAppTiers.value.dapps.forEach((dapp) => {
+        const tierId = dapp.tierId!;
+        tierCounts[tierId] = (tierCounts[tierId] || 0) + 1;
+      });
+
+      // Convert the tierCounts object to a sorted array of counts
+      const sortedCounts = Object.entries(tierCounts)
+        .sort((a, b) => Number(a[0]) - Number(b[0]))
+        .map((entry) => entry[1]);
+
+      return sortedCounts;
+    });
+
+    const tokensToBeBurned = computed(() => {
+      // Calculate the sum of tokens to be burned
+      const tbb = dAppTiers.value.rewards.reduce((acc: bigint, reward: BigInt, i) => {
+        const slotsPerTier = tiersConfiguration.value.slotsPerTier[i];
+        const dappsInTier = dappsPerTier.value[i];
+        const tokensForTier =
+          ((BigInt(reward.toString()) * (BigInt(slotsPerTier) - BigInt(dappsInTier))) /
```

Comment:
`reward` is BigInt, no need to convert

Reply:
The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.ts(2362)
Operator '*' cannot be applied to types 'BigInt' and 'bigint'.ts(2365)
(parameter) reward: BigInt

Reply:
You have issues because of this `(acc: bigint, reward: BigInt)` 
`bigint` is a build type and `BigInt` is a function which create a value of `bigint` type.

Replace  with `acc: bigint, reward: bigint`

Reply:
fixed